### ISSUE TITLE
Icon request: tiktok (tik-tok) [Released in 5.13.1]

### ISSUE THREAD

**Issue by patternbased** [FortAwesome/Font-Awesome#14340]
### What can it be used for?

to link to ones web based tiktok page. example: http://vm.tiktok.com/JMwccQ/

### Example image (optional)

you can see the log in the tiktok app or here: http://vm.tiktok.com/JMwccQ/

### Icon request checklist

- [Single] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [X ] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [X ] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [XLink ] I have included at least one example image if it could be helpful (optional)
- [X ] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by Akagilnc** [comment]
any update for this issue?

**Comment by thewizardplusplus** [comment]
As a temporary solution, I can suggest using my works:

Version | Icon | Link
--- | --- | ---
base | ![base_64x64](https://user-images.githubusercontent.com/3732121/54728031-4e240980-4b8c-11e9-92d7-edb5f70188e5.png) | https://openclipart.org/detail/317125/tiktok-icon
double | ![double_64x64](https://user-images.githubusercontent.com/3732121/54728036-59773500-4b8c-11e9-827e-0d449c20373a.png) | https://openclipart.org/detail/317126/tiktok-icon
contour | ![contour_64x64](https://user-images.githubusercontent.com/3732121/54728042-63993380-4b8c-11e9-9839-b71692359ca8.png) | https://openclipart.org/detail/317127/tiktok-icon

License is [Creative Commons Zero 1.0 Public Domain Dedication (CC0 1.0)](https://creativecommons.org/publicdomain/zero/1.0/), i.e. they are released into the public domain.

**Comment by tagliala** [comment]
@Akagilnc sorry for the late reply.

Recently, we have added a [faqs section](https://fontawesome.com/community/leaderboard/brands#faqs) at the bottom of our [Brand Leaderboard](https://fontawesome.com/community/leaderboard/brands). There you will find a lot of information on how requests work

**Comment by zomilanovic** [comment]
+1

**Comment by zomilanovic** [comment]
On a side note I would prefer it to become `tiktok` instead of `tik-tok`

**Comment by Daniel-Griffiths** [comment]
Another +1 from me. It would be handy to have this in!

In our current app we have to use a custom svg for tiktok since it's one of the few social networks missing from font awesome.

**Comment by steve231293** [comment]
Any update on this, i need this icon too.

**Comment by nickdesignz** [comment]
I've my own svg file. Hopefully it helps you guys out till Font-Awesome made it. 

[Tik Tok Icon SVG](https://github.com/FortAwesome/Font-Awesome/files/3505000/tik-tok-icon-black.svg.zip)


)

**Comment by xFirez** [comment]
Bump?

**Comment by jeromelive33** [comment]
2019: We need a tiktok icon 😁😁😁😁

**Comment by lvphantoms** [comment]
I would also like to throw my hat into the request bin on this one. We could certainly use this icon!

**Comment by aprmack** [comment]
I signed up for github just so I can ask for a TikTok icon. Please make this!

**Comment by ex-avenger** [comment]
Any updates regarding the same?

**Comment by nathanbrauer** [comment]
TikTok's MOBILE website currently has its mark available as a simple SVG:

```svg

<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48">
    <g fill="none" fill-rule="evenodd">
        <path fill="#00F2EA" d="M20.023 18.111v-1.703a13.17 13.17 0 0 0-1.784-.13c-7.3 0-13.239 5.94-13.239 13.24 0 4.478 2.238 8.442 5.652 10.839a13.187 13.187 0 0 1-3.555-9.014c0-7.196 5.77-13.064 12.926-13.232"/>
        <path fill="#00F2EA" d="M20.335 37.389c3.257 0 5.914-2.591 6.035-5.82l.011-28.825h5.266a9.999 9.999 0 0 1-.17-1.825h-7.192l-.012 28.826c-.12 3.228-2.778 5.818-6.034 5.818a6.006 6.006 0 0 1-2.805-.694 6.037 6.037 0 0 0 4.901 2.52M41.484 12.528v-1.602a9.943 9.943 0 0 1-5.449-1.62 10.011 10.011 0 0 0 5.45 3.222"/>
        <path fill="#FF004F" d="M36.035 9.305a9.962 9.962 0 0 1-2.461-6.56h-1.927a10.025 10.025 0 0 0 4.388 6.56M18.239 23.471a6.053 6.053 0 0 0-6.046 6.046 6.05 6.05 0 0 0 3.24 5.352 6.007 6.007 0 0 1-1.144-3.526 6.053 6.053 0 0 1 6.046-6.047c.623 0 1.22.103 1.784.28v-7.343a13.17 13.17 0 0 0-1.784-.13c-.105 0-.208.006-.312.008v5.64a5.944 5.944 0 0 0-1.784-.28"/>
        <path fill="#FF004F" d="M41.484 12.528v5.59c-3.73 0-7.185-1.193-10.007-3.218v14.617c0 7.3-5.938 13.239-13.238 13.239-2.821 0-5.437-.89-7.587-2.4a13.201 13.201 0 0 0 9.683 4.225c7.3 0 13.239-5.939 13.239-13.238V16.726a17.107 17.107 0 0 0 10.007 3.218V12.75c-.72 0-1.42-.078-2.097-.223"/>
        <path fill="#FFF" d="M31.477 29.517V14.9a17.103 17.103 0 0 0 10.007 3.218v-5.59a10.011 10.011 0 0 1-5.449-3.223 10.025 10.025 0 0 1-4.388-6.56h-5.266L26.37 31.57c-.121 3.228-2.778 5.819-6.035 5.819a6.038 6.038 0 0 1-4.901-2.52 6.05 6.05 0 0 1-3.241-5.352 6.053 6.053 0 0 1 6.046-6.046c.622 0 1.219.102 1.784.28v-5.64c-7.156.168-12.926 6.036-12.926 13.232 0 3.48 1.352 6.648 3.555 9.014a13.16 13.16 0 0 0 7.587 2.399c7.3 0 13.238-5.939 13.238-13.239"/>
    </g>
</svg>
```

**Comment by SjorsHartwijk** [comment]
+1

**Comment by hollygirouard** [comment]
+1 Please! Seems like this is a growing trend.

**Comment by freddygreve** [comment]
You can use this code to implement TikTok to FA:
```html
<i class="fab fa-tiktok"></i>

<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/js/all.min.js"></script>
<script type="text/javascript">
    var faTikTok = {
        prefix: 'fab',
        iconName: 'tiktok',
        icon: [24, 24, [], 'e001', 'M12.53.02C13.84 0 15.14.01 16.44 0c.08 1.53.63 3.09 1.75 4.17 1.12 1.11 2.7 1.62 4.24 1.79v4.03c-1.44-.05-2.89-.35-4.2-.97-.57-.26-1.1-.59-1.62-.93-.01 2.92.01 5.84-.02 8.75-.08 1.4-.54 2.79-1.35 3.94-1.31 1.92-3.58 3.17-5.91 3.21-1.43.08-2.86-.31-4.08-1.03-2.02-1.19-3.44-3.37-3.65-5.71-.02-.5-.03-1-.01-1.49.18-1.9 1.12-3.72 2.58-4.96 1.66-1.44 3.98-2.13 6.15-1.72.02 1.48-.04 2.96-.04 4.44-.99-.32-2.15-.23-3.02.37-.63.41-1.11 1.04-1.36 1.75-.21.51-.15 1.07-.14 1.61.24 1.64 1.82 3.02 3.5 2.87 1.12-.01 2.19-.66 2.77-1.61.19-.33.4-.67.41-1.06.1-1.79.06-3.57.07-5.36.01-4.03-.01-8.05.02-12.07z']
    }

    FontAwesome.library.add(
        faTikTok
    )
</script>
```

**Comment by woorise** [comment]
+1 on TikTok

**Comment by bubbaspaarx** [comment]
+1 for tiktok please

**Comment by boomtemp123** [comment]
+1

**Comment by borjagiron** [comment]
I want it too. Thanks

**Comment by OTt8** [comment]
Me too please. Thanks!

**Comment by oliverjessner** [comment]
+1

**Comment by suciusergiu94** [comment]
Come on guys, give us the Icon or some explanation :(

**Comment by fabiomn** [comment]
2020 and still no icon is available!

**Comment by fabiomn** [comment]
> 2019: We need a tiktok icon 😁😁😁😁

2020 and still no icon is available!

**Comment by s3739** [comment]
They care not about this matter?

**Comment by pixelloop** [comment]
How is this still not part of the icon pack? :o

**Comment by lewisdonovan** [comment]
+1

**Comment by oliverjessner** [comment]
+1

**Comment by mikejoseph** [comment]
+1

**Comment by bobo23** [comment]
+1

**Comment by ThibaultG4U** [comment]
This would be a great and needed addition!

**Comment by ChristianAziz** [comment]
+1

**Comment by ChristianAziz** [comment]
This would make FontAwesome.. awesomer!

**Comment by Rich-Core** [comment]
+1 for this. It's one of the most popular apps in the world right now. We need this logo!

**Comment by xcaq** [comment]
> You can use this code to implement TikTok to FA:
> 
> ```
> <i class="fab fa-tiktok"></i>
> 
> <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/js/all.min.js"></script>
> <script type="text/javascript">
>     var faTikTok = {
>         prefix: 'fab',
>         iconName: 'tiktok',
>         icon: [24, 24, [], 'e001', 'M12.53.02C13.84 0 15.14.01 16.44 0c.08 1.53.63 3.09 1.75 4.17 1.12 1.11 2.7 1.62 4.24 1.79v4.03c-1.44-.05-2.89-.35-4.2-.97-.57-.26-1.1-.59-1.62-.93-.01 2.92.01 5.84-.02 8.75-.08 1.4-.54 2.79-1.35 3.94-1.31 1.92-3.58 3.17-5.91 3.21-1.43.08-2.86-.31-4.08-1.03-2.02-1.19-3.44-3.37-3.65-5.71-.02-.5-.03-1-.01-1.49.18-1.9 1.12-3.72 2.58-4.96 1.66-1.44 3.98-2.13 6.15-1.72.02 1.48-.04 2.96-.04 4.44-.99-.32-2.15-.23-3.02.37-.63.41-1.11 1.04-1.36 1.75-.21.51-.15 1.07-.14 1.61.24 1.64 1.82 3.02 3.5 2.87 1.12-.01 2.19-.66 2.77-1.61.19-.33.4-.67.41-1.06.1-1.79.06-3.57.07-5.36.01-4.03-.01-8.05.02-12.07z']
>     }
> 
>     FontAwesome.library.add(
>         faTikTok
>     )
> </script>
> ```

thanks a lot ❤️

**Comment by Lokinorse** [comment]
Omg... Opened ticket since 2018...

**Comment by nickdesignz** [comment]
> Omg... Opened request since 2018...

It's weird isn't it? It looks that they don't want to create it :(

**Comment by oliverjessner** [comment]
@nickdesignz @lokinorse relax they are already working on it

**Comment by MAnfal** [comment]
Hey guys any ideas on when it will be available?

**Comment by bensti22** [comment]
what is tiktok?

**Comment by asadullazadeh** [comment]
I wait since 2018 and I still believe you are do it 😄

**Comment by rodri8** [comment]
@freddygreve 
Thanks for the code, but its not working for me when I'm using the font awesome kits
for example:
`<script src="https://kit.fontawesome.com/xxxxxxxxxx.js" crossorigin="anonymous"></script>`


> You can use this code to implement TikTok to FA:
> 
> ```
> <i class="fab fa-tiktok"></i>
> 
> <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/js/all.min.js"></script>
> <script type="text/javascript">
>     var faTikTok = {
>         prefix: 'fab',
>         iconName: 'tiktok',
>         icon: [24, 24, [], 'e001', 'M12.53.02C13.84 0 15.14.01 16.44 0c.08 1.53.63 3.09 1.75 4.17 1.12 1.11 2.7 1.62 4.24 1.79v4.03c-1.44-.05-2.89-.35-4.2-.97-.57-.26-1.1-.59-1.62-.93-.01 2.92.01 5.84-.02 8.75-.08 1.4-.54 2.79-1.35 3.94-1.31 1.92-3.58 3.17-5.91 3.21-1.43.08-2.86-.31-4.08-1.03-2.02-1.19-3.44-3.37-3.65-5.71-.02-.5-.03-1-.01-1.49.18-1.9 1.12-3.72 2.58-4.96 1.66-1.44 3.98-2.13 6.15-1.72.02 1.48-.04 2.96-.04 4.44-.99-.32-2.15-.23-3.02.37-.63.41-1.11 1.04-1.36 1.75-.21.51-.15 1.07-.14 1.61.24 1.64 1.82 3.02 3.5 2.87 1.12-.01 2.19-.66 2.77-1.61.19-.33.4-.67.41-1.06.1-1.79.06-3.57.07-5.36.01-4.03-.01-8.05.02-12.07z']
>     }
> 
>     FontAwesome.library.add(
>         faTikTok
>     )
> </script>
> ```

**Comment by cristopherrosenberg** [comment]
Still no tik-tok icon by now??

**Comment by sensibleworld** [comment]
It's coming in our next release.

**Comment by bradshroyer** [comment]
Unfortunately, it's a necessary evil now. Good to see it's coming with the next release.

**Comment by tetuaoro** [comment]
...2020 june.. hahaha I search it too, well

**Comment by tagliala** [comment]
Hi,

Thanks for your patience.

Tik-Tok is available in the just released 5.13.1 🚀

https://fontawesome.com/icons/tiktok

@sensibleworld Apologies, I took the honor of closing this request by myself 😅

**Comment by supmanyu** [comment]
Hi,

**Comment by SolutionSpirit** [comment]
User this SVG and it will work.

`<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="tiktok" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="nav_icons text-dark svg-inline--fa fa-tiktok fa-w-14"><path fill="currentColor" d="M448,209.91a210.06,210.06,0,0,1-122.77-39.25V349.38A162.55,162.55,0,1,1,185,188.31V278.2a74.62,74.62,0,1,0,52.23,71.18V0l88,0a121.18,121.18,0,0,0,1.86,22.17h0A122.18,122.18,0,0,0,381,102.39a121.43,121.43,0,0,0,67,20.14Z" class=""></path></svg>`

Enjoy :)

**Comment by scottie** [comment]
What is going on ? Why is this so hard for you to do ?

**Comment by tagliala** [comment]
Hi @scottie 

This icon has been released in FA 5.13.1, please upgrade

https://fontawesome.com/v5.15/icons/tiktok?style=brands
